source/schemas/types.txt

```diff
@@ -509,3 +509,80 @@ with the ``title`` field set to the embedded object's type name:
             }
       }
    }
+
+.. _geospatial:
+
+Geospatial Data
+---------------
+
+Geospatial data describes points and other data on the earth's surface. App 
+Services supports defining GeoJSON Point data in a schema. 
+
+GeoJSON Point
+~~~~~~~~~~~~~
+
+A GeoJSON Point is a single location on the Earth's surface, described by its 
+longitude and latitude. To ensure wide compatibility, App Services stores these 
+points in accordance to the `GeoJSON spec <https://datatracker.ietf.org/doc/html/rfc7946>`_.
+To store a GeoPoint in Atlas, you must follow these rules:
```

Comment:
nit - these aren't really requirements for storing in Atlas so much as:

- a schema/definition of a GeoJSON Point in general
- requirements for how you use Point objects in your Sync models (i.e. "It must be embedded within another type")

For the "embedded within another type" requirement, I think we can be clearer that the limitation is specifically if you want to query the `Point` field with geospatial RQL operators. You could totally have a regular object type called `Point` that follows this schema and it would save & sync just fine, right?

Reply:
yeah, I guess it may be nice to introduce what the JSON schema for a GeoJSON Point would look like perhaps before diving into how to structure the MongoDB documents themselves (unless I'm misunderstanding your comment). I think the main thing we need to make clear here is that the Atlas Device Sync will only recognize a property as a GeoJSON Point if the JSON schema meets the proper embedded object structure. The structure of the documents themselves should follow this schema, and are subject to additional constraints (noted below in the comment below / in this doc [here](https://docs.google.com/document/d/10aAwBVwWRl2OOiG8RCXfT_-ejIWe6KmYFtrZgGPW15k/edit#heading=h.fgl0tx27vwoj))